Day14_YH

Yeong-Huns opened the pull request:
## 📌 본인이 생각하는 문제 난이도 
- [ ] 상
- [ ] 중
- [x] 하

<br>

## 📌 중점적으로 리뷰를 받고 싶은 파트
> 💡 가까운 수

<br>

## 📌 하고 싶은 말
> 💡 스트림으로 풀고 싶었는데 잘 생각이 안나네 좋은 의견있으면 달아줘

m-jeon commented on Day14/Day14_YH.md:
```diff
@@ -0,0 +1,245 @@
+        System.out.println(min);
+        System.out.println(rs);
+        return rs;
+    }
```
import java.util.*; 없어도 될꺼같음

Kimjonghyeon6084 replied:
다들 비슷하게 푼 거 신기함

m-jeon reviewed (approved):
확인완료

Kimjonghyeon6084 commented on Day14/Day14_YH.md:
```diff
@@ -0,0 +1,245 @@
+            .filter(i->"369".contains(i))
+            .collect(Collectors.joining(""))
+            .length();
+            //스트림이 쓰고 싶었어..
```
스트림 중독이네 이 친궄ㅋㅋㅋ
나 같이 프젝할 때 스트림 안쓴다고 욕먹을 거 같은데 조졌다..

Kimjonghyeon6084 commented on Day14/Day14_YH.md:
```diff
@@ -0,0 +1,245 @@
+class Solution {
+    public String solution(String my_string) {
+    return Arrays.stream(my_string.split(""))
+        .map(i->i.equals(i.toUpperCase()) ? i.toLowerCase() : i.toUpperCase())
```
이야 깔끔하게 잘햇네

Kimjonghyeon6084 reviewed (approved):
아주 굿입니당

rapier4231 reviewed (approved):
스트림 👍   ㅎ ㅏ지만 난 1번이 이해가 안되었쯰 👍

Yeong-Huns commented:
> 스트림 👍 ㅎ ㅏ지만 난 1번이 이해가 안되었쯰 👍
@rapier4231 @NeDongSem  
😋
```java
       Arrays.stream((order+"") // (정수+"")-> 문자열로 변환
            .split("")) -> 문자열을 .split("")으로 한자리씩 나눔 ex) "54321" -> {"5", "4", "3", "2", "1"}  (리턴타입 String[] )
            .filter(i->"369".contains(i)) //-> "문자열".contains(전달인자) : 문자열에  전달인자가 포함되어있으면true,아니면 false 
            .collect(Collectors.joining("")) // Collectors.joining("") or Collectors.joining() -> 하나의 문자열로 만듬 (String[] -> String)
            .length(); //-> 문자열의 길이
```
